.github/workflows/lint.yaml

```
@@ -0,0 +1,21 @@
+name: Lint
+on:
+  push:
```

Let's add some spacing here for the top-level keys.

```suggestion
name: Lint

on:
  push:
  
```